src/content/changes/cluster-apps-for-aws/cluster-aws/v0.76.0-alpha.1.md

```diff
@@ -0,0 +1,18 @@
+---
+# Generated by scripts/aggregate-changelogs. WARNING: Manual edits to this files will be overwritten.
+changes_categories:
+- Cluster apps for AWS
+changes_entry:
+  repository: giantswarm/cluster-aws
+  url: https://github.com/giantswarm/cluster-aws/releases/tag/v0.76.0-alpha.1
```

Comment:
It's still here after re-creating this PR, even though we removed both the alpha tag and release. Locally, this one doesn't get generated for me. Maybe we can just ignore it for now and merge the changes 🤷  @pipo02mix?